### DISCUSSION TITLE
Announcement: Officially announcing that Announcements™ will be announced here!

### DISCUSSION THREAD

**Discussion by LiamSwayne** [LiamSwayne/Elratio#5] [Announcements]
As we take this great leap forward, we notice that we are gracefully moving backwards, back to a time of family values. We here at Elratio HQ, short for Half-witted Quitters, have decided to make announcements to engage with our community. But first, a moment of silence.

<pre>

</pre>

Damn. That's a lotta silence.

**Comment by JonathanBout** [comment]
‎ ‎ ‎ 
‎ ‎ 
‎ 
‎ 
‎ 
‎ 
‎

**Comment by Dhruv00000** [comment]
That silence needs to be filled with the empty keyword.
